lang/system/pl.xml

```diff
@@ -40,11 +88,22 @@
 			<level>Poziom</level>
 			<name>Nazwa</name>
 			<trophy_earning>Zdobywanie trofeów</trophy_earning>
-			<parental_controls>kontrola rodzicielska</parental_controls>
+			<parental_controls>Kontrola rodzicielska</parental_controls>
```

Comment:
Can you check it ? @Croden1999

Reply:
He is right.

Reply:
@Zangetsu38 All I did is match to rest strings displayed in this section aka start capitalized.

Reply:
ok, it is fine, no idea why have error here